### PR TITLE
G33 minor change

### PR DESCRIPTION
**Pull request by LVD-AC**
- further named remaining tests
- A and O  parameters obsolete
- quick homing with home_delta()

### CODE REVIEW

File: Marlin/Marlin_main.cpp [hunk 1]
```diff
@@ -5103,16 +5099,20 @@ void home_all_axes() { gcode_G28(); }
...
 
+      stepper.synchronize();
       #if HAS_LEVELING
-        set_bed_leveling_enabled(false);
+        reset_bed_level; //after calibration bed-level will no longer be valid
       #endif
-
-      home_all_axes();
+      #if HOTENDS > 1
+        const uint8_t old_tool_index = active_extruder;
```
Comment (LVD-AC) [inline Marlin/Marlin_main.cpp]: this line is obsolete

File: Marlin/Marlin_main.cpp [hunk 2]
```diff
@@ -5103,16 +5099,20 @@ void home_all_axes() { gcode_G28(); }
...
-      const char negating_parameter = do_height_only ? 'A' : do_center_and_towers ? 'O' : 'T';
-      int8_t probe_mode = code_seen(negating_parameter) && code_value_bool() ? -probe_points : probe_points;
+      bool towers_set = (code_seen('T') ? code_value_bool() : true);
 
       SERIAL_PROTOCOLLNPGM("G33 Auto Calibrate");
 
+      stepper.synchronize();
       #if HAS_LEVELING
-        set_bed_leveling_enabled(false);
+        reset_bed_level; //after calibration bed-level will no longer be valid
```
Comment (thinkyhead) [inline Marlin/Marlin_main.cpp]: What about `reset_bed_level`? To call it, add parentheses `()`.

File: Marlin/Marlin_main.cpp (hunk 1 above)
Comment (thinkyhead) [reply Marlin/Marlin_main.cpp]: You have my leave to remove it.

File: Marlin/Marlin_main.cpp (hunk 2 above)
Comment (LVD-AC) [reply Marlin/Marlin_main.cpp]: ok, i'll do so tomorrow

File: Marlin/Marlin_main.cpp [hunk 3]
```diff
@@ -5412,11 +5411,14 @@ void home_all_axes() { gcode_G28(); }
         }
 
         stepper.synchronize();
-
-        home_all_axes();
+        endstops.enable(true);
```
Comment (thinkyhead) [inline Marlin/Marlin_main.cpp]: `setup_for_endstop_or_probe_move` perhaps here? And after homing, `clean_up_after_endstop_or_probe_move`… possibly… See if these functions do the right thing for this situation.

Comment (thinkyhead) [reply Marlin/Marlin_main.cpp]: In general, after enabling endstops to do something, when done the method to call is `endstops.not_homing()` which will set them back to their global non-homing state.

**Comment by LVD-AC** [comment]
Squashed and cleaned in #6621

**Pull request closed without merging**